### PR TITLE
Ticket 1

### PR DESCRIPTION
**Pull request by rezazad68**
panda added successfuly

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import numpy as np
 import pandas as pd
 
+
```
Comment (rezazad68) [inline main.py:4]: remove empty lines

File: main.py [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+
```
Comment (rezazad68) [inline main.py:5]: test

File: main.py (hunk 1 above)
Comment (ISIC-CVPR20) [inline main.py:4]: remove lines

**Review by rezazad68 — COMMENTED** [review]
done

**Review by ISIC-CVPR20 — APPROVED** [review]

**Pull request merged**